docs: update Python helloworld tutorial for a2a-sdk v1.0 (#1775)
a2aproject/A2A: docs/tutorials/python/6-interact-with-server.md

    --8<-- "https://raw.githubusercontent.com/a2aproject/a2a-samples/refs/heads/main/samples/python/agents/helloworld/test_client.py:A2ACardResolver"
    ```

    The `A2ACardResolver` class is a convenience. When `get_agent_card()` is called, it fetches the `AgentCard` from the server's `/.well-known/agent-card.json` endpoint (based on the provided base URL), which is then used to initialize the client.

2. **Initializing the Client & Sending a Non-Streaming Message**:

    ```python { .no-copy }
    --8<-- "https://raw.githubusercontent.com/a2aproject/a2a-samples/refs/heads/main/samples/python/agents/helloworld/test_client.py:message_send"
    ```

>     - `create_client` is an async factory function that creates a `Client` configured from the fetched `AgentCard` and a `ClientConfig`. Here `streaming=False` selects the non-streaming transport.

Reviewer: ```suggestion
    - The `create_client` function creates a `Client` based on the information provided by `AgentCard` and a `ClientConfig`.
```

    - We construct a `Message` using the `new_text_message` helper (passing `role=Role.ROLE_USER`), then wrap it in a `SendMessageRequest`.

>     - The client's `send_message` method returns an async iterable that yields a single final `Task` response from the agent.

Reviewer: ```suggestion
    - The client's `send_message` method returns an async iterator object. In this example, it likely has only one final `Task` response from the agent.
```
